package Protos;

import "base.proto";

message SlidingLtdC {
    required double power = 1;
    required double modulation = 2;
    required double target_rate = 3;
    required double tau_mean = 4;
    required double min_weight = 5;
    required double max_weight = 6;
}

message SlidingLtdState {
    required double p_mean = 1;
}

